source/src/microapp/cs_Microapp.cpp

```diff
@@ -212,12 +231,13 @@ void Microapp::resetState(uint8_t index) {
 }
 
 void Microapp::resetTestState(uint8_t index) {
```

Comment:
misses `.started`

Reply:
It should not be unstarted on enable, else the app will try to register callbacks again etc.